Server/JokesOnYou.Web.Api/Services/JwtTokenService.cs

```diff
@@ -13,11 +14,10 @@ public class JwtTokenService : ITokenService
         private readonly JwtSecurityTokenHandler _tokenHandler;
         private readonly byte[] _key;
 
-        public JwtTokenService()
+        public JwtTokenService(IConfiguration config)
```

Comment:
It's not a bid deal, but I think it's a bit of overhead to pass the whole configuration to the constructor when we only need a key.

Reply:
We can ofc bind to a class but then we have one class with one property, is there another approach?